packages/js/src/Modules/Verto/webrtc/BaseCall.ts

```diff
@@ -44,12 +44,45 @@ import Call from './Call';
  * @ignore Hide in docs output
  */
 export default abstract class BaseCall implements IWebRTCCall {
+  /**
+   * The identifier of the call.
+   * It becomes `callID` to send to server.
```

Comment:
```suggestion
   * The call identifier.
```

I don't think we need to explain that `callId` is sent to the server, since Call Control doesn't really get that info